sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubsScaleMonitor.cs

```diff
@@ -89,8 +89,8 @@ public async Task<EventHubsTriggerMetrics> GetMetricsAsync()
 
             try
             {
-                checkpoints = await Task.WhenAll(checkpointTasks)
-                    .ConfigureAwait(false);
+                checkpoints = (await Task.WhenAll(checkpointTasks).ConfigureAwait(false))
```

Comment:
Hmm.. could Task.WhenAll actually return null when awaited?

Reply:
It can, but only in the case where the partition information reports no partitions.   Under normal circumstances, this should not occur but is possible in some odd corner cases.

Reply:
actually... I'm incorrect.   Even when there are no tasks to await, it seems that it emits an empty array.   I'll update to remove the guard.